cmd/rest/client.go

```diff
@@ -75,6 +75,7 @@ func (n *NetworkError) Unwrap() error {
 // Client - http based RPC client.
 type Client struct {
 	connected int32 // ref: https://golang.org/pkg/sync/atomic/#pkg-note-BUG
+	lastConn  int64
```

Comment:
See the bug noted above. This will have alignment problems on some platforms.
```suggestion
	_           int32
	lastConn    int64
```
This should AFAICT make it align.

Reply:
> This should AFAICT make it align.

Done, yes it looks like it needs 64 bits alignment